### ISSUE TITLE
mapControls and defaultLocation not working

### ISSUE THREAD

**Issue by EldenChris** [cheesegrits/filament-google-maps#7]
I can't get any of the mapControls and the defaultLocation to work.

These options are just ignored no matted what I do, any idea on how to fix this ?

**Comment by hossam-hani** [comment]
i have the same issue

**Comment by cheesegrits** [comment]
@EldenChris @hossam-hani can one/both of you drop by the #google-maps channel in the Filament Discord to discuss this?

**Comment by glkrenx** [comment]
I also experienced the same problem. If I use the resource admin panel, there's no problem, but if I use a custom page, I have a problem that I can't use defaultLocation. After I looked into it, it turns out there's a problem when I use form fill with data:
$this->form->fill([
'title' => $this->post->title,
'content' => $this->post->content,
]);

But when I don't send data when calling form fill, then defaultLocation works.
